### ISSUE TITLE
[MJWMUT] Mutants! - Module Crash

### ISSUE THREAD

**Issue by tuday2** [enusbaum/MBBSEmu#101]
**Module Information**
* Identifier: MJWMUT
* Module Name: Mutants!
* Version: 3.11

**Describe the bug**
Module will crash with error message

**To Reproduce**
1. Load Mutants! module
2. Play game, kill monsters

**Expected behavior**
Module runs without crashing

**Screenshots**
```
Critical Exception has occurred:
System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter 'Unknown Exported Function Ordinal in MAJORBBS: 339')
   at MBBSEmu.HostProcess.ExportedModules.Majorbbs.Invoke(UInt16 ordinal, Boolean offsetsOnly)
   at MBBSEmu.HostProcess.ExecutionUnits.ExecutionUnit.ExternalFunctionDelegate(UInt16 ordinal, UInt16 functionOrdinal)
   at MBBSEmu.CPU.CpuCore.Tick()
   at MBBSEmu.HostProcess.ExecutionUnits.ExecutionUnit.Execute(IntPtr16 entryPoint, UInt16 channelNumber, Boolean simulateCallFar, Boolean bypassState, Queue`1 initialStackValues, UInt16 initialStackPointer)
   at MBBSEmu.Module.MbbsModule.Execute(IntPtr16 entryPoint, UInt16 channelNumber, Boolean simulateCallFar, Boolean bypassSetState, Queue`1 initialStackValues, UInt16 initialStackPointer)
   at MBBSEmu.HostProcess.MbbsHost.AddModule(MbbsModule module)
   at MBBSEmu.Program.Run(String[] args)
```

**Software Information:**
- OS: Windows Server 2019 Standard
- Telnet Client: Netrunner

**Comment by enusbaum** [comment]
Ordinal `MAJORBBS.339` is `hexopt` which retrieves a hex option from the specified MCV file. 

![image](https://user-images.githubusercontent.com/1139047/92723246-d2e18500-f336-11ea-865a-520551fe742f.png)

Best I can tell at the moment, it just returns a 1 byte value (0-255), or between the boundaries specified.

**Comment by enusbaum** [comment]
While this will be fixed by #122 , it's Mutants! (MJWMUT) will still need `ACCBB` as it tries to look up the Mutants! Link user. This is the same issue as #113 

We can confirm the fix here and leave #113 open for the `ACCBB` fix.
